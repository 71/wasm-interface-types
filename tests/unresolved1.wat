;; parse-fail

(module
  (@interface func (type $foo)))

(; CHECK-ALL:
failed to find type named `$foo`
     --> tests/unresolved1.wat:4:26
      |
    4 |   (@interface func (type $foo)))
      |                          ^
;)
