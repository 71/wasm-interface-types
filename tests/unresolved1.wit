;; parse-fail

(module
  (@interface func (type $foo)))
