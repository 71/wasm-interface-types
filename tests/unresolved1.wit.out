failed to find type named `$foo`
     --> ../../tests/unresolved1.wit:4:26
      |
    4 |   (@interface func (type $foo)))
      |                          ^
